contracts/governance/FeeSharingProxy/FeeSharingLogic.sol

```diff
@@ -340,21 +405,20 @@ contract FeeSharingLogic is SafeMath96, IFeeSharingProxy, Ownable, FeeSharingPro
             return (0, 0);
         }
 
-        uint256 start = processedCheckpoints[_user][_loanPoolToken];
+        uint256 processedCheckpoints = processedCheckpoints[_user][_loanPoolToken];
```

Comment:
shadowing declaration  
```suggestion  
        uint256 processedUserCheckpoints = processedCheckpoints[_user][_loanPoolToken];  
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/458/commits/0a1fc81b156a2db3f06da8e6bd9b91670bb02a71)